content/en/hosting/monitoring/setup.md

```diff
@@ -141,19 +141,36 @@ With the [release of 1.1.0](https://github.com/medic/cht-watchdog/releases/tag/1
 
-#### couch2pg Data (Remote)
+#### CHT Sync Data (Remote)
 
-While not the default setup, and not what most deployments need, you may want to set up a way to monitor couch2pg data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
+While not the default setup, and not what most deployments need, you may want to set up a way to monitor CHT Sync data without sharing any Postgres credentials. Instead of sharing credentials, you expose an HTTP endpoint that requires no login or password.  Of course, similar to  CHT Core's [Monitoring API]({{< relref "apps/reference/api#get-apiv2monitoring" >}}), this endpoint should be configured to not share sensitive information (since it will be publicly accessible).
+
+If you are running CHT Sync using Kubernetes:
+1. Uncomment the `metrics_exporter` value in your `values.yaml` file and make sure `enabled` is set to `true`.
+1. Expose the metrics endpoint to the internet. This can be done by setting the `service.type` to `LoadBalancer` or `NodePort` in your `values.yaml` file.
+1. On your watchdog instance, create a custom scrape definition file: `cp exporters/postgres/scrape.yml ./exporters/postgres/scrape-custom.yml`
+1. Edit `scrape-custom.yml` so that it has the ip address of `SQL_EXPORTER_IP` from step 2 above.  If that was `10.220.249.15`, then you file would look like:
+   ```yaml
+    scrape_configs:
+      - job_name: sql_exporter
+        static_configs:
+          - targets: ['10.220.249.15:9399']
+   ```
+1. Finally, on your watchdog instance, start (or restart) your server including the `compose.scrape-only.yml` compose file:
+    ```bash
+    docker compose --env-file .env  -f exporters/postgres/compose.yml -f exporters/postgres/compose.scrape-only.yml up -d
+    ```
 
 To run a remote instance of only the SQL Exporter on your Postgres server:
 
 1. Clone this repo: `git clone git@github.com:medic/cht-watchdog.git` and `cd` into `cht-watchdog`
 2. Copy `exporters/postgres/sql_servers_example.yml` to `exporters/postgres/sql_servers.yml`
 3. Edit the new `exporters/postgres/sql_servers.yml` file to have the correct credentials for your server. You need to update the `USERNAME` and `PASSWORD`.   You may need to update the IP address and port also, but likely the default values are correct.
 4. Copy `.env.example` to `.env` 
-5. In the new `.env` file, edit `SQL_EXPORTER_IP` to be public IP of the Posgtres server
+5. In the new `.env` file, edit `SQL_EXPORTER_IP` to be the public IP of the Posgtres server
 6. Start the service with these two compose files*:
    ```shell
    docker compose --env-file .env  -f exporters/postgres/compose.yml -f exporters/postgres/compose.stand-alone.yml up -d
```

Comment:
I can't make a suggestion for code you didn't commit, but first cut off these off at step 7 and add something like:

```
Continue on to next section to [set up the scrape on Watchdog](#set-up-watchdog-to-scrape-the-sql-exporter).
```

Then renumber step 8 here to start at 1.  Add a title and some content below that.  Something like:

```
##### Set up Watchdog to scrape the SQL Exporter

No matter which way you set up your SQL exporter, follow these steps to tell your Watchdog instance to scrape the new endpoint.
```
```